models/player.py

```diff
@@ -0,0 +1,48 @@
+from .game import Game
+from .board import Board
+
+class Player(Game):
```

Comment:
why pass Game to the Player?

Reply:
I want to access _roll_ and _endgame_ in Player. 
P.S. After going through your solution, I realized that I have mixed up models and services. If I could split them, this irrelevant import.